app/domain/synchronize/mailchimp/result.rb

```diff
@@ -46,30 +48,37 @@ def badge_info
         [state, STATE_BADGES[state]]
       end
 
+      def forgotten_emails
+        Array(Array(data.dig(:subscribe_members, :failed)).last).map do |op|
```

Comment:
Wow dieses dig + last war schwierig zu verstehen. Bis ich da mal nachvollziehen konnte wo überall diese Datenstruktur zusammengestellt wird... Aber müsste funktionieren.

Reply:
habs noch leicht umgestellt und in eine methode extrahiert und kommentiert, aber ist einfach nicht ideal, die datenstruktur, wollte aber nicht noch grundlegender umstellen